<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSchemeEditor</class>
 <widget class="QWidget" name="ColorSchemeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>376</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="discriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="colorTable"/>
   </item>
   <item>
    <widget class="QCheckBox" name="randomizedBackgroundCheck">
     <property name="toolTip">
      <string>Hue and saturation values of default foreground and background colors are randomized by default. Some color schemes might use different randomization settings.
To see any effect, set colors with saturation value greater than 0.</string>
     </property>
     <property name="text">
      <string>Randomly adjust colors for each session</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="blurCheckBox">
     <property name="text">
      <string>Blur background</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="transparencyLabel">
       <property name="text">
        <string>Background transparency:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="transparencySlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="transparencyPercentLabel">
       <property name="text">
        <string>Percent</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="wallpaperLabel">
       <property name="text">
        <string>Background image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wallpaperPath">
       <property name="toolTip">
        <string>Edit the path of the background image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="wallpaperSelectButton">
       <property name="toolTip">
        <string>Choose the background image</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>364</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="KMessageWidget" name="transparencyWarningWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
